src/parts/_meter_progress.css

```diff
@@ -0,0 +1,58 @@
+#meter-wrapper {

```

Comment:
From `README.md`:
> Water.css is a css framework that doesn't require any classes.

It isn't explicit, but we wouldn't require any IDs in the HTML content either. The idea is that you can add water.css to your page without having to make any changes. This would require having a `#meter-wrapper` added to the page, so it doesn't meet the goal of the project.